### PR TITLE
Fix database routers

### PR DESCRIPTION
**Pull request by thepsalmist**
## Description

Restore database router settings, omitted in #124

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: sensorsafrica/router.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from django.conf import settings
 
 class ReplicaRouter:
-    read_replicas = settings.READ_DATABASE_URLS
+    read_replicas = list(settings.DATABASES.keys())
```
Comment (kilemensi) [inline sensorsafrica/router.py:6]:
```suggestion
    databases = list(settings.DATABASES.keys())
```

I take it this includes **_everything_** i.e. `default` + all `read_replica_xxx`.

**Review by kilemensi — APPROVED** [review]
🚀 

---

If router was disabled, how did we test https://github.com/CodeForAfrica/sensors.AFRICA-api/pull/122?

File: sensorsafrica/router.py (hunk 1 above)
Comment (thepsalmist) [reply sensorsafrica/router.py:6]: Yes this includes all the databases @kilemensi

Comment (kilemensi) [reply sensorsafrica/router.py:6]: Shouldn't call it `read_replicas` then @thepsalmist

**Pull request merged**